po/fr.po

```diff
@@ -135,7 +137,7 @@ msgstr "Alphabétique"
 #: ui/prefs-gtk4.ui:15 ui/prefs.ui:18
 msgctxt "folder-position"
 msgid "Start"
-msgstr "Démarrer"
+msgstr "Début"
```

Comment:
This is a value of the setting whether folders should be located at the start or end of the app grid, or in alphabetical order mixed with other applications. So I think "Début" is the word to use here.
Does "fin" need to be changed then as well, or is this OK (my French is a bit rusty, sadly...)?

Reply:
Oui, « Début » is correct here. I don’t understand what you want to change « Fin » to. « Fin » is the correct opposite of « Début » and translation for “End”.

Reply:
Merci ;)